webapp/lib/controller.dart

```diff
@@ -211,7 +212,7 @@ void listenForDriverMetrics(String project, List<String> drivers) {
   // start listening for the new project collection
   driverMetricsSubscriptions.forEach((subscription) => subscription?.cancel());
   driverMetricsSubscriptions.clear();
-  for (var driver in drivers) {
+  for (var driver in drivers[selectedProject]) {
     driversDataMap[driver] = [];
     driverMetricsSubscriptions.add(platform.listenForMetrics(
       'projects/$selectedProject/driver_metrics/$driver/metrics',
```

Comment:
Looks like this slipped through previously - shouldn't it be:
```suggestion
      'projects/$project/driver_metrics/$driver/metrics',
```
?

Reply:
Oh yes. I agree